<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_option">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>120</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QPushButton" name="btn_play">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_suspend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Suspend</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_replay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Replay</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_next">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Shape</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="win_score">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdScore">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
